### DISCUSSION TITLE
Skinning Addon/Icons with Masque

### DISCUSSION THREAD

**Discussion by rulezxy** [SFX-WoW/Masque#117] [Questions] [answered]
Since the last update/s its not possible to Skin Icons from Addons like Doom_Cooldown_Pulse (DCP).
Is it possible to add this option again? Since you can skin the whole UI (Icons) it should be possible to skin Icons using the default skins.

**Comment by StormFX** [comment]
I'm not sure what you mean. DCP has to support Masque, not the other way around.

**Comment by rulezxy** [comment]
I dont understand why it isnt working anymore. A few months ago it showed up (in Masque) and you where able to skin it. 

DCP doesnt changed anything related to that so I cant find the issue. 

Is there a different from classic to retail because I cant find anything in the code. 

...otherwise I will just ask at dcp's author.

**Comment by StormFX** [comment]
Honestly, I'd never even heard of DCP until you mentioned it. Nothing in Masque has changed that should cause it to stop working altogether. I'll look at it if I get time, but I've got quite a bit of other stuff to do before 9.0.

**Comment by rulezxy** [comment]
Thank you for your time and help.

Its just a optional addon that gives short information when a buff/ability is rdy or runs out. 

I had it for a long time but to some issues with the filter system I disabled it (before the updates). 

I try to let dcp check masque on load but even if I do write this profile manually (code) it doesnt skin the icon.

Anyways its just optical and shadowlands is coming soon so I will just wait for a dcp update.

**Comment by StormFX** [comment, marked as answer]
I just looked at the code and there is no Masque support in DCP.

**Comment by rulezxy** [comment]
Ok, I will leave it to dcp then / setting up a weakaura instead. Ty :)
